apollo-normalized-cache/build.gradle.kts

```diff
@@ -33,22 +33,19 @@ kotlin {
     }
 
     val jvmMain by getting {
-      dependsOn(commonMain)
```

Comment:
I never really understood why we did this. The Kotlin plugin does this automatically, right? Or did this have unwanted side effects?

Reply:
Yeah I guess it was just a legacy